apsuite/asparams.py

```diff
@@ -0,0 +1,35 @@
+"""Nominal Accelerator Structure Parameters."""
+
+# These are nominal accelerator structure parameters conveniently stored
+# in this module. Some parameters are optics-related and should be updated
+# if any lattice changes.
+
+RF_FREQ = 499666000  # [Hz]
+
+BO_RING_LEN = 496.78745  # [m]
+BO_HARM_NR = 828
+BO_REV_FREQ = RF_FREQ / BO_HARM_NR
+BO_REV_TIME = BO_HARM_NR / RF_FREQ
+BO_NR_BPMS = 50
+BO_TUNEX = 19.20433
+BO_TUNEY = 7.31417
+BO_MOM_COMPACT = 7.166e-04
+
+SI_RING_LEN = 518.3899  # [m]
+SI_HARM_NR = 864
```

Comment:
I used what seemed to be the most prevalent abbreviation in the repo, `_NR_`. But I do not have a particular preference. Can we standardize to `_NUM_`, @murilobalves ?

Reply:
I do not have any preference on `NR` or `NUM` (this is actually a problem because in that way I end up not following a convention). Once you choose one here, I can propagate the convention to standardize apsuite classes (many of them are using `nr` and others `num`).

Reply:
done, @fernandohds564 !